docs/api.rst

```diff
@@ -2001,6 +2001,48 @@ Getting all available counterparties
     :statuscode 200: Success
     :statuscode 500: Internal rotki error
 
+Getting all supported chains
+==============================
+
+.. http:get:: /api/(version)/blockchains/supported
+
+    Doing a GET on the supported chains will return all supported chains along with their type
+
+    **Example Request**
+
+    .. http:example:: curl wget httpie python-requests
+
+    GET /api/(version)/blockchains/supported HTTP/1.1
+    Host: localhost:5042
+    Content-Type: application/json;charset=UTF-8
+
+    {}
+
+    **Example Response**
+
+    .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Content-Type: application/json
+
+        {
+            "result": [
+                {"name": "ETH", "type": "evm"},
+                {"name": "OPTIMISM", "type": "evm"},
+                {"name": "AVAX", "type": "evm"},
+                {"name": "ETH2", "type": "eth2"},
+                {"name": "DOT", "type": "substrate"},
+                {"name": "KSM", "type": "substrate"},
+                {"name": "BTC", "type": "bitcoin"},
+                {"name": "BCH", "type": "bitcoin"}
+            ]
+            "message": ""
+        }
+
+    :resjson object result: Contains all supported chains and their types
```

Comment:
we tend to add a `.` at the end of each row but can be changed later (I have to do a PR for the docs yet)

Reply:
sure